### ISSUE TITLE
Somfy motors - Not really an issue, for now :)

### ISSUE THREAD

**Issue by luckyzor** [evgeni/esphome-somfy#16]
Hello Evgeni,

I came accross this repo, amazing btw, because I will install somfy motors at home and I want to avoid their hub with the IO motors.
I ask to install the RTS motors, but I'm wondering if this project still works with new motors, if this is a reliable solution in 2023.

thanks for you help

**Comment by evgeni** [comment]
I don't think you can get new RTS motors in 2023 and the io stuff is completely different

**Comment by luckyzor** [comment]
I talked to my installer and I can choose between RTS or IO motors.
RTS are a bit cheaper.

**Comment by luckyzor** [comment]
don't know if they are new models or not.

**Comment by evgeni** [comment]
Are there any other (non Somfy) solutions available?

My main concern is that the solution here seems not to work with *all* RTS devices and if you end up with one that are not compatible, you got some old stuff that nobody is trying to get working anymore.
